### DISCUSSION TITLE
How to open DOM inspector for local deployment on PC

### DISCUSSION THREAD

**Discussion by 277584579** [rustdesk/rustdesk#9905] [Q&A]
am a beginner and started locally using the sciter.dll method according to the documentation, but I am unable to open DOM inspector to debug styles. Do you have any relevant documentation on how to solve this problem?

**Comment by rustdesk** [comment]
https://github.com/rustdesk/rustdesk/discussions/763, go to [discord](https://discord.gg/nDceKgxnkV) please, search there first, some users might have asked similar questions. Usually nobody will help you with build or self-hosting here.
